### ISSUE TITLE
Can't launch itch.io version from Steam (Linux)

### ISSUE THREAD

**Issue by rbits0** [EverestAPI/Everest#743]
### My mods and Everest install are up to date

Yes

### I have recreated the bug with only Everest OR a minimum number of mods enabled

Yes

### Describe the bug

I have installed Everest to my itch.io copy of Celeste. When I open the "Celeste" executable it works, but I have added the executable to Steam and it won't launch. Doesn't matter if I have any mods enabled. It also doesn't generate any log.txt in the Celeste folder if I do this. It used to work, so I assume an Everest update broke it.

### Steps to reproduce

- Install Celeste from itch.io
- Install latest Everest using Olympus
- Add Celeste to Steam
- Click Play

### Expected behavior

Launches the game.

### Operating System

KDE Neon 5.27

### Everest Version

4607

**Comment by Kalobi** [comment]
Maybe Steam is trying to launch it with Proton?

**Comment by rbits0** [comment]
No, "Force the use of a specific Steam Play compatibility tool" under the Compatibility tab of properties is disabled, so it should be running it normally, without Proton. Also I haven't changed anything in Steam relating to Celeste since it was working.

**Comment by Kalobi** [comment]
Do you definitely have the same executable selected in Steam that you use for manually starting the game? There's both the `Celeste` shell script and a native executable.

**Comment by rbits0** [comment]
Yep. I don't see a shell script in the Celeste folder, the path in Steam is /home/riley/.config/itch/apps/celeste/Celeste which is what I run to successfully launch it outside of Steam.

When I uninstall Everest there is a shell script and an executable, and both work fine when launching from Steam.
